### PR TITLE
Update RC1 & RC2 bundles

### PR DESCRIPTION
**Pull request by collivier**
It also fixes the RA1 one.

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

**Comment by collivier** [comment]
please review it as it has to be cherry-picked to stable/lakelse right after.

**Review by gkunz — CHANGES_REQUESTED** [review]
Wasn't the release moved to January https://wiki.anuket.io/display/HOME/Lakelse+Schedule ?

**Comment by collivier** [comment]
@gkunz  I follow RA1 and RA2 as RC1 and RC2 depend on them - We could update all of them instead if they do match the marketing timeframe.

**Comment by gkunz** [comment]
> @gkunz I follow RA1 and RA2 as RC1 and RC2 depend on them - We could update all of them instead if they do match the marketing timeframe.

Ah, I didn't mean the bundle version, but the release date. Sorry, could have marked that in the files directly. It states December 7, but the sign-off date has been moved to Jan 4, 2022 according to the release schedule.

File: doc/ref_arch/openstack/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This is OpenStack based Reference Architecture (RA-1)
 
 **Version: _0_**
 
-**Release Date: _1st July 2021_**
+**Release Date: _7th December 2021_**
```
Comment (gkunz) [inline doc/ref_arch/openstack/README.md:11]: Example according to https://wiki.anuket.io/display/HOME/Lakelse+Schedule

```suggestion
**Release Date: _4th January 2022_**
```

**Comment by collivier** [comment]
Why not. but even now, we don't know if milestones would move just before the release (it happened more than once).
We do fix RA1 and RA2 too.

File: doc/ref_arch/openstack/README.md (hunk 1 above)
Comment (collivier) [reply doc/ref_arch/openstack/README.md]: done

**Comment by collivier** [comment]
@gkunz it's updated - thank you in advance for your review.

**Review by karinesevilla — APPROVED** [review]

**Review by gkunz — APPROVED** [review]
thanks for the update

**Review by pgoyal01 — APPROVED** [review]

**Pull request merged**